tools/infrastructure/docker/QNX/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Add local user
+# Either use the LOCAL_USER_ID if passed in at runtime or
+# fallback
+
+USER_ID=${LOCAL_USER_ID:-9001}
+
+EX_CODE=$(id -u developer > /dev/null 2&>1 || echo $?)
+if [[ $EX_CODE -ne $? ]]; then
+    useradd --shell /bin/bash -u $USER_ID -o -c "" developer
+    echo "developer ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+    chown -R developer:developer /home/developer
+    export HOME=/home/developer
+fi
+
+echo "Welcome to developer_qnx:build!"
```

Comment:
@alignment alignment if you want

Reply:
@ZhdanovP fixed within ae7d52ab0f457d8f46b1ce7aedaf336c3f9d1933